electron-react/src/electron-wait-react.js

```diff
@@ -1,7 +1,8 @@
 const net = require("net");
 const port = process.env.PORT ? process.env.PORT - 100 : 3000;
+const config = require("./util/config");
 
-process.env.ELECTRON_START_URL = `http://localhost:${port}`;
+process.env.ELECTRON_START_URL = `http://${config.self_hostname}:${port}`;
```

Comment:
This should be of interest to @freddiecoleman so I want to make sure he's had a look at this too.

Reply:
Yeah this was motivated by his docker use case, now there is only one place to change your IP (config.yaml)

Reply:
Just noticed this.

I run ran dev branch in a container and got it to work after changing the `config.yaml`.

Thank you.